docs/getting-started.md

```diff
@@ -286,19 +290,24 @@ kubectl describe ee virtualmachine-i-0033eb4a6c846451d -n sample-ns
 # Output
 Name:         virtualmachine-i-0033eb4a6c846451d
 Namespace:    sample-ns
-Labels:       environment.tag.nephe=nephe
-              kind.nephe=virtualmachine
-              login.tag.nephe=ubuntu
-              name.nephe=i-0033eb4a6c846451d
-              name.tag.nephe=vpc-0d6bb6a4a880bd9ad-ubuntu1
-              namespace.nephe=sample-ns
-              terraform.tag.nephe=true
-              vpc.nephe=vpc-0d6bb6a4a880bd9ad
+Labels:       nephe.io/cloud-region=us-west-1
+              nephe.io/cloud-vm-name=vpc-0d6bb6a4a880bd9ad-ubuntu1
+              nephe.io/cloud-vm-uid=i-0033eb4a6c846451d
+              nephe.io/cloud-vpc-name=test-us-west1-vpc
+              nephe.io/cloud-vpc-uid=vpc-0d6bb6a4a880bd9ad
+              nephe.io/environment-tag=nephe
+              nephe.io/kind=virtualmachine
+              nephe.io/login-tag=ec2-user
+              nephe.io/name-tag=vpc-0d6bb6a4a880bd9ad-ubuntu1
+              nephe.io/namespace=sample-ns
+              nephe.io/terraform-tag=true
+              nephe.io/vm-name=i-0033eb4a6c846451d
```

Comment:
rename it to nephe.io/owner-vm-name and nephe.io/onwer-vm-vpc

Reply:
changed vm-name to owner-vm and vpc to owner-vm-vpc